js/src/predicate.ts

```diff
@@ -77,10 +77,9 @@ export class Col<T= any> extends Value<T> {
 
 export abstract class Predicate {
     abstract bind(batch: RecordBatch): PredicateFunc;
-    and(expr: Predicate): Predicate { return new And(this, expr); }
-    or(expr: Predicate): Predicate { return new Or(this, expr); }
+    and(...expr: Predicate[]): And { return new And(this, ...expr); }
+    or(...expr: Predicate[]): And { return new Or(this, ...expr); }
```

Comment:
Should the return type here be `: Or`?

Reply:
wow, not sure how I got away with that mistake